state_machine/applications/flight/lib/radio_utils/commands.py

```diff
@@ -212,6 +245,9 @@ def file_exists(path):
     DELETE_FILE: {"function": delete_file, "name": "DELETE_FILE", "will_respond": True, "has_args": True},
     RELOAD: {"function": reload, "name": "RELOAD", "will_respond": True, "has_args": False},
     REQUEST_BEACON: {"function": request_beacon, "name": "REQUEST_BEACON", "will_respond": True, "has_args": False},
+    GET_RTC: {"function": get_rtc, "name": "GET_RTC", "will_respond": True, "has_args": False},
+    SET_RTC: {"function": set_rtc_utime, "name": "SET_RTC", "will_respond": True, "has_args": True},

```

Comment:
since this expects a unix timestamp, I'd say the command name should be `SET_RTC_UTIME` to match your convention elsewhere.

Reply:
fair